import_export_extensions/admin/model_admins/export_job_admin.py

```diff
@@ -42,6 +42,13 @@ class ExportJobAdmin(
         "cancel_jobs",
     )
 
+    def get_form(self, request: WSGIRequest, obj=None, **kwargs):
+        """Push admin site to form."""
+        form = super().get_form(request, obj, **kwargs)
+
+        form.admin_site = self.admin_site
+        return form
```

Comment:
Is it really required? Why?

Please try to answer questions `why?` rather than `what does it do?` in docstring

We also made some changes to the admin mixins, so this logic can probably be put into `BaseCeleryImportExportAdminMixin`. Please check the latest changes when you get back to work